src/fidesops/schemas/saas/saas_config.py

```diff
@@ -193,6 +194,16 @@ class SaaSConfig(BaseModel):
     test_request: SaaSRequest
     data_protection_request: Optional[SaaSRequest] = None  # GDPR Delete
 
+    @validator("connector_params")
+    def validate_connector_params(cls, v: List[ConnectorParam]) -> List[ConnectorParam]:
```

Comment:
I was including this since we were talking about potentially using the domain to identify a SaaS connector. I got ahead of myself and agree with you that we should give this more thought, rolling back for now.